packages/frontend/src/layout/HomePage/HomePage.scss

```diff
@@ -134,3 +134,9 @@ $LightGrey: #b2b2b2;
   box-sizing: border-box;
   padding: 10px 20px;
 }
+
+.portal-switch{
+  &:focus-visible{
+    outline:3px solid #000000;
```

Comment:
A few minor stylistic nits:
1. Add a space between the colon and the "3px"
2. When a hex code involves repeated numbers, by convention we can shorten it to a 3-digit abbreviation. So instead of `#000000` we can say `#000`

So in the end this is what it would look like:
```
outline: 3px solid #000;
```